gradle/publishable-java-library.gradle

```diff
@@ -0,0 +1,46 @@
+def licenseFiles = copySpec {
+    //mockito license
+    from(".") { include 'LICENSE' }
+}
+
+//TODO why do we still ship the main jar with sources?
```

Comment:
The use case was for Eclipse users that manually dropped mockito jar on the classpath. They would have automatic source for debugging / javadoc out of the box. It didn't help IDEA out of the box because IDEA does not look for sources inside jar. This was the time before declarative dependency resolution where people kept jars checked-in.

We should move on and stop putting sources in main jar. Nobody does it this way in the modern world. Makes the jar bigger and has some other issues.